src/midi.js

```diff
@@ -22,19 +31,25 @@ const midi = (notes, fileName) => {
 		// only the first noteOn (or noteOff) needs the complete arity of the function call
 		// subsequent calls need only the first 2 args (channel and note)
 		if (noteObj.note) {
+			var note = noteObj.note[0];
+			//Get the octave identifier (2 for a2, 5 for e5)
+			var oct = parseInt(note.slice(1,note.length));
```

Comment:
To make sure all features are fully transposed, I also added functionality to transpose all scales generated by scribble.mode.